README.md

```diff
@@ -58,6 +58,29 @@ To build Mixxx, run
 There should now be a `mixxx` executable in the current directory that you can
 run. Alternatively, can generate a package using `cpack`.
 
+### Using Dev Container
+
+> [!NOTE]
+> Dev container has recently been introduced and it is likely incomplete! Currently, it's been tested with `devpod` on Zed and Codium exclusively
+
+We provide a DevContainer definition for Mixxx. The container base uses Ubuntu 24.04 to ensure a close similarity with the CI.
+Similar to Fedora Toolbox's, the default configuration provide a comprehensive definition of mounts which will allow use to use Mixxx seamlessly in the container. (UI, audio, ...)
```

Comment:
Why reference  "Fedora Toolbox's"

Reply:
Because this is the other big initiative around dev containerisation. It's less mature, supported and popular (yet), but I know we discuss about it as well so I thought it was worth to make the parallel for user that my have heard only of Fedora Toolbox (e.g Silverblue user) since it literally equivalent and work as a drop-in replacement

Reply:
```suggestion
The default configuration provide a comprehensive definition of mounts which will allow you to use Mixxx seamlessly in the container. (UI, audio, ...)
```

Reply:
OK, in that case I would add an additional sentences that explains it the rationals and implications ... or link external resources.